### DISCUSSION TITLE
Option to add MIME type

### DISCUSSION THREAD

**Discussion by hjelmdal** [gtm-support/vue-gtm#347] [Ideas]
Hi, 
I would love to see the option to add MIME Type to the script that is injected, so that you can configure for ie. Partytown etc. For that I would need to add type="text/partytown" to the script. 

Thanks in advance :-)

**Comment by vladimir-chernykh** [comment]
Hey!

I also bumped into the same problem and created a PR [here](https://github.com/gtm-support/core/pull/350). When (if) it gets merged, adding MIME type should be easy.
